.travis.yml

```diff
@@ -6,14 +6,7 @@ cache:
   directories:
     - $HOME/.composer/cache/files
 
-## PHP versions we test against
-php:
-  - 7.0
-  - 7.1
-  - nightly
-  - hhvm
-
-## Build matrix for lowest and highest possible targets
```

Comment:
Travis must have changed something since the way original matrix is set up, the composer lowest, current, and highest possible are not tested as the matrix is overriding with the specified php versions.